daemon/graphdriver/windows/windows.go

```diff
@@ -779,8 +779,8 @@ func writeLayerReexec() {
 }
 
 // writeLayer writes a layer from a tar file.
-func writeLayer(layerData io.Reader, home string, id string, parentLayerPaths ...string) (int64, error) {
-	err := winio.EnableProcessPrivileges([]string{winio.SeBackupPrivilege, winio.SeRestorePrivilege})
+func writeLayer(layerData io.Reader, home string, id string, parentLayerPaths ...string) (size int64, err error) {
+	err = winio.EnableProcessPrivileges([]string{winio.SeBackupPrivilege, winio.SeRestorePrivilege})
```

Comment:
@jhowardmsft would renaming the `err` output variable to `retErr` address your concerns?

I know we've had cases where the output variable was overlooked, and naming it `retErr` instead of plain `err` makes it a bit more apparent that it's an output variable

Reply:
I like that change. Changed to use `retErr` to prevent confusion.